test/round-test.js

```diff
@@ -83,61 +85,53 @@ describe('Round', function() {
         });
 
         it('should count turns', function() {
-            takeTurn('sea otter', round);
+            takeTurn('Roald Dahl', round);
 
             expect(round.turns).to.equal(1);
         });
 
         it('should only count the turn if a valid answer choice is guessed', function() {
-            takeTurn('sea otter', round);
+            takeTurn('Roald Dahl', round);
             takeTurn(null, round);
 
             expect(round.turns).to.equal(1);
             expect(round.incorrectGuesses).to.deep.equal([])
         });
 
         it('should set the next card in the deck as the current card after a turn', function() {
-            let turn1 = takeTurn('sea otter', round);
-            let turn2 = takeTurn('spleen', round);
+            let turn1 = takeTurn('Roald Dahl', round);
+            let turn2 = takeTurn('Errol', round);
             let invalidTurn = takeTurn(null, round);
-            let turn3 = takeTurn('Lex', round);
+            let turn3 = takeTurn('Middle-earth', round);
 
             expect(round.currentCard).to.deep.equal(card4)
             expect(turn3).to.equal('Incorrect!');
             expect(round.turns).to.equal(3);
-            expect(round.incorrectGuesses).to.deep.equal([14, 12])
+            expect(round.incorrectGuesses).to.deep.equal([2, 3])
         });
     });
-    
+
     describe('Calculate Percent Correct', function() {
```

Comment:
Look into if there's a way to use the beforeEach hook to declare these two sets of turn variables for calculatePercentCorrect and endRound without nesting those two blocks inside of another `describe` block?